src/lib_ccx/ccx_encoders_helpers.c

```diff
@@ -235,49 +235,50 @@ unsigned char *close_tag(struct encoder_ctx *ctx, unsigned char *buffer, char *t
 
 unsigned get_decoder_line_encoded(struct encoder_ctx *ctx, unsigned char *buffer, int line_num, struct eia608_screen *data)
 {
-	int col = COL_WHITE;
+	int colour = COL_WHITE;
```

Comment:
Small thing but since this is new... we have "color" in some places. Let's stick to one spelling. Since CC means "closed captions" which is an American term let's use the American spelling, i.e. "color". 
Not saying it's the correct one but it was in the code first.

Reply:
Yeah when I put that in I hadn't seen any other code to follow